opcodes/riscv-opc.c

```diff
@@ -761,13 +762,14 @@ const struct riscv_opcode riscv_opcodes[] =
 {"c.fswsp",   32, INSN_CLASS_F_AND_C, "CT,CM(Cc)", MATCH_C_FSWSP, MASK_C_FSWSP, match_opcode, INSN_DREF|INSN_4_BYTE },
 {"c.fsw",     32, INSN_CLASS_F_AND_C, "CD,Ck(Cs)", MATCH_C_FSW, MASK_C_FSW, match_opcode, INSN_DREF|INSN_4_BYTE },
 
-/* ZCEE subset, where ZCEE is a subset of ZCEA  */
-{"c.sext.b",   0, INSN_CLASS_ZCEE,  "Cs",  MATCH_C_SEXT_B, MASK_C_SEXT_B, match_opcode, 0 },
-{"c.sext.h",   0, INSN_CLASS_ZCEE,  "Cs",  MATCH_C_SEXT_H, MASK_C_SEXT_H, match_opcode, 0 },
-{"c.zext.b",   0, INSN_CLASS_ZCEE,  "Cs",  MATCH_C_ZEXT_B, MASK_C_ZEXT_B, match_opcode, 0 },
-{"c.zext.h",   0, INSN_CLASS_ZCEE,  "Cs",  MATCH_C_ZEXT_H, MASK_C_ZEXT_H, match_opcode, 0 },
-{"c.zext.w",  64, INSN_CLASS_ZCEE,  "Cs",  MATCH_C_ZEXT_W, MASK_C_ZEXT_W, match_opcode, 0 },
-{"c.mul",      0, INSN_CLASS_ZCEE,  "Cs,Ct",  MATCH_C_MUL, MASK_C_MUL, match_opcode, 0 },
+/* ZCEE subset, where ZCEE is a subset of ZCE  */
+{"c.sext.b",  0,  INSN_CLASS_ZCEE,  "CZc",  MATCH_C_SEXT_B, MASK_C_SEXT_B, match_opcode, 0 },
+{"c.sext.h",  0,  INSN_CLASS_ZCEE,  "CZc",  MATCH_C_SEXT_H, MASK_C_SEXT_H, match_opcode, 0 },
+{"c.zext.b",  0,  INSN_CLASS_ZCEE,  "CZc",  MATCH_C_ZEXT_B, MASK_C_ZEXT_B, match_opcode, 0 },
+{"c.zext.h",  0,  INSN_CLASS_ZCEE,  "CZc",  MATCH_C_ZEXT_H, MASK_C_ZEXT_H, match_opcode, 0 },
+{"c.sext.w",  64, INSN_CLASS_ZCEE,  "nd",   MATCH_C_ADDIW, MASK_C_ADDIW|MASK_RVC_IMM, match_opcode, INSN_ALIAS },
```

Comment:
Note that c.sext.w will be a pseudo-instruction for c.addiw rd, 0, so I think it is not need to add here